releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,20 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 6.0.0 - November 29, 2022
+* Fixed a bug where the configuration did not refresh when ChainedConfigurationProvider was used. [#168](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/168)
+* Added the following new API for additional App Configuration geo-replication support. [#385](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/385)
+
+   ```cs
+   public AzureAppConfigurationOptions Connect(IEnumerable<string> connectionStrings)
+   ```
+
+   The new API allows you to provide an ordered list of connection strings of your App Configuration store and its replicas.
+
+* Removed `IConfigurationRefresher.SetDirty` API. `IConfigurationRefresher.ProcessPushNotification` API should be used for push-model based configuration refresh. Refer to [this tutorial](https://docs.microsoft.com/en-us/azure/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh?tabs=windowscommandprompt) for more details about the `ProcessPushNotification` API. [#357](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/357)
```

Comment:
It will be good to explain why it's removed.

Reply:
Is it enough to mention the reason in the original issue, for "necessary data consistency" provided by ProcessPushNotification?